guides/common/modules/proc_viewing-booted-container-images.adoc

```diff
@@ -0,0 +1,19 @@
+[id="Viewing_booted_container_images_{context}"]
+= Viewing booted container images
+
+These actions streamline the management of image mode hosts, providing flexibility and control over host operations through {Project}.
+
+.Prerequisites
+* *Pull-Mode Remote Execution*: If you use pull-mode remote execution, the `x`, `y`, and `z` packages must be included in the image mode host's container image.
+For more information, see xref:transport-modes-for-remote-execution_{context}[].
+
+.Procedure
+. On the new Hosts Overview page, navigate to *Content* > *Booted Container Images* to view all images used by image mode hosts.
+This page displays the spread of digests under specific image paths.
+More digests under an image path indicate greater drift in the host ecosystem, ideally minimized by ensuring all hosts run the most up-to-date image version.
+. Click the host count to navigate to the *All Hosts Listing* page, which identifies the hosts associated with each specific image and displays the number of hosts using each digest.
+. On a host's *Details* tab, use the image mode *Details* card to see the current `bootc` status.
+This card includes a link to the *Bootc Action - Script Default* remote execution job.
```

Comment:
While this is true, it may be more helpful to be user-focused here. "To perform a bootc action on the host, click the xxx link."